static/js/brand-store/Members/Members.js

```diff
@@ -71,6 +76,10 @@ function Members() {
     dispatch(fetchInvites(id));
   }, [id]);
 
+  useEffect(() => {
+    setFilteredMembers(members);
```

Comment:
Typically you want to avoid having two sources of truth for a dataset.

What can potentially happen here is that you filter a set of members below, but then the member list updates which will then trigger this and wipe out the filter. What you probably want to do is have a conditional that checks if there is a filter and if not then use the full list, else use the filtered results.